bootstrap_test.py

```diff
@@ -280,6 +280,28 @@ def test_consistent_range_movement_true_with_rf1_should_fail(self):
     def test_consistent_range_movement_false_with_rf1_should_succeed(self):
         self._bootstrap_test_with_replica_down(False, rf=1)
 
+    def test_RF_GT_Nodes_should_succeed(self):
```

Comment:
is having multiple DCs essential for this problem?
If so then maybe it should be reflected in the test name?

Reply:
Yep multi-dc is the key. I can change the name.